1.1/translations/sf_fr.ttl

```diff
@@ -0,0 +1,171 @@
+c) Deux Anneaux de la frontière ne se croisent pas et les Anneaux de la frontière d'un Polygone ne peuvent se croiser en un Point, mais uniquement en tant que tangente. 
+d) Un polygone ne peut pas avoir de lignes de coupe, de pointes ou de perforations. 
+e) L'intérieur de chaque polygone est un ensemble de points connexe.
+f) L'extérieur d'un polygone avec 1 ou plusieurs trous n'est pas connecté. Chaque trou définit un composant connecté de l'extérieur."""@fr  .
+
+sf:PolyhedralSurface 
+    skos:prefLabel "Surface Polyédrique"@fr ;
+    skos:definition """Une géométrie sf:PolyhedralSurface est une collection contiguë de sf:Polygon, qui partagent des segments de frontière communs. Pour chaque paire de polygones qui se touchent, la limite commune doit pouvoir être exprimée comme une collection finie de sf:LineString. Chacune de ces sf:LineString doit faire partie de la limite d'au plus de 2 pièces sf:Polygon. 
+
+Pour deux polygones quelconques partageant une limite commune, le sommet du sf:Polygon doit être cohérent. Cela signifie que lorsque deux anneaux linéaires de ces deux sf:Polygon traversent le segment de frontière commun, ils le font dans des directions opposées. Puisque la surface polyédrique est contiguë, tous les sf:Polygon seront donc orientés de manière cohérente. Cela signifie qu'une surface non orientée (telle que la bande Mbius) ne doit pas avoir de représentations de surface unique. Ils peuvent être représentés par une sf:MultiSurface. 
+
+Si chacune de ces sf:LineString est la limite d'exactement de 2 pièces sf:Polygon, alors sf:PolyhedralSurface est un simple polyèdre fermé et est topologiquement isomorphe à la surface d'une sphère. Par le théorème de surface de Jordan (théorème de Jordan pour les 2 sphères), de tels polyèdres renferment un solide topologiquement isomorphe à l'intérieur d'une sphère; le ballon. Dans ce cas, le haut de la surface pointera soit vers l'intérieur, soit vers l'extérieur du solide fini fermé. Si vers l'extérieur, la surface est la limite extérieure de la surface fermée. Si vers l'intérieur, la surface est l'intérieur du complément infini du solide enfermé. Une boule avec un certain nombre de vides (trous) à l'intérieur peut donc être présentée comme une coque limite extérieure et un certain nombre de coques limites intérieures. """@fr .
+
+sf:Surface
+    skos:prefLabel "Surface"@fr ;
+    skos:definition """Une géométrie sf:Surface est un objet géométrique à 2 dimensions. Un sf:Surface simple peut consister en d'une pièce unique associé à une frontière extérieure et à 0 ou plusieurs frontières intérieures. Une seul pièce de surface dans un espace tridimensionnel est isométrique aux surfaces planes, par une simple matrice de rotation affine qui fait tourner la pièce sur le plan z = 0. Si la pièce n'est pas verticale, la projection sur le même plan est un isomorphisme, et peut être représenté comme une transformation linéaire, c'est-à-dire une affine. 
```

Comment:
"peut consister en une pièce unique associée" drop the d' and turn the verb to feminine

Reply:
rewrote.